Algebras/DotBracket/Parts/algpart_dotBracket_basic.gap

```diff
@@ -5,13 +5,29 @@
     return res;
   }
 
+  string sadd_cut(Subsequence lb, Subsequence region, string e) {
```

Comment:
may I recommend to move those new algebra functions into a new file "algpart_cofold_dotBracket.gap" and include it from the alg_dotBracket.gap file. This convention would better fit into the existing naming convention.